docs/specs/supporting-tools.md

```diff
@@ -0,0 +1,65 @@
+# Supporting tools and services
+
+This page outlines tools and services that currently support the development container specification, including the `devcontainer.json` format. A `devcontainer.json` file in your project tells tools and services that support the dev container spec how to access (or create) a dev container with a well-defined tool and runtime stack. 
+
+While most [dev container properties](devcontainerjson-reference.md) apply to any supporting tool or service, a few are specific to certain tools, which are outlined below.
+
+## Dev container CLI
+
+There will be a dev container command line interface (CLI) that can take a `devcontainer.json` and create and configure a dev container from it.
+
+The publishing of this CLI is being discussed in an [issue](https://github.com/microsoft/dev-container-spec/issues/9).
+
+## GitHub Codespaces
+
+A [codespace](https://docs.github.com/en/codespaces/overview) is a development environment that's hosted in the cloud. Codespaces run on a variety of VM-based compute options hosted by GitHub.com, which you can configure from 2 core machines up to 32 core machines. You can connect to your codespaces from the browser or locally using Visual Studio Code.
+
+> **Tip:** If you've already built a codespace and connected to it, be sure to run **Codespaces: Rebuild Container** from the Command Palette (`kbstyle(F1)`) to pick up any changes you make.
+
+### Product specific limitations
```

Comment:
Given the location of Codespaces, I wonder if it wouldn't make sense to mention that VS Code specific properties are supported by the Codespaces web editor in addition to VS Code client.

Reply:
Good call out. Updated to `Please note that Codespaces, both through the Codespaces web editor and VS Code client, and supports [VS Code properties](#visual-studio-code-remote---containers).`, let me know if that sounds good to you.

Reply:
Maybe move it above the heading? Looks like a limitation.  Might make sense to just put a product specific properties heading here and include it there. I know there's some in the works already so having that section probably generally makes sense.  For now what goes under that heading could just be, "GitHub Codespaces works with a growing number of tools and, where applicable, their devcontainer.json properties. For example, connecting the Codespaces web editor or VS Code enables the use of [VS Code properties](#visual-studio-code-remote---containers)."

Reply:
Makes sense - added `Product specific properties` with the content you mention. Let me know what you think.

Reply:
Makes sense!